### PR TITLE
More UI fixes for index page

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: elements-AE/search-screen.html [hunk 1]
```diff
@@ -33,8 +33,8 @@
                 <force-sobject-sync sobject="{{sobject}}" fieldstoindex="{{searchfields}}" query="{{syncQuery}}" on-sync-progress="handleSyncResponse" on-sync-complete="handleSyncResponse"></force-sobject-sync>
                 <force-sobject-collection id="list" sobject="{{sobject}}" querytype="[[searchtype]]" query="[[searchquery]]" collection="{{searchresults}}" cache-promise="[[cachePromise]]"></force-sobject-collection>
                 <!--<iron-list id="uilist" height="{{resultHeight}}" items="[[models]]" as="model" on-touchend="stopClick" on-tap="window.location.hash="#edit/"+model.id;">-->
```
Comment (bhariharan) [inline elements-AE/search-screen.html]: Can we remove the commented out line?

**Comment by bhariharan** [comment]
Looks good otherwise.

**Comment by kchitalia** [comment]
Done :)

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**